.circleci/config.yml

```diff
@@ -176,24 +138,24 @@ jobs:
           path: reports/junit
       - run:
           name: "Upload coverage"
-          command: cd .. && bash <(curl -s https://codecov.io/bash) -F dapp -C $CIRCLE_SHA1
+          command: bash <(curl -s https://codecov.io/bash) -F dapp -C $CIRCLE_SHA1
 
   build_dapp:
     executor: base-executor
-    working_directory: *dapp_working_dir
+    working_directory: *working_dir
     steps:
       - attach_workspace: *attach_options
       - run:
           name: Build dapp
           command: >
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              npm run build -- --mode ghpages;
+              pnpm run build --filter raiden-dapp -- --mode ghpages;
             else
-              npm run build -- --mode development;
+              pnpm run build --filter raiden-dapp -- --mode development;
             fi
       - run:
           name: Compress dapp
-          command: tar -czvf dapp.tar.gz dist
+          command: tar -czvf dapp.tar.gz raiden-dapp/dist
```

Comment:
This won't work. The way `tar` works, it'll create the file with a `/raiden-dapp/dist` folder inside, while the `serve_pr` tool expects it to have `/dist` on the root